src/main/java/uk/gov/ons/ctp/response/action/state/ActionSvcStateTransitionManagerFactory.java

```diff
@@ -119,6 +119,11 @@ public ActionSvcStateTransitionManagerFactory() {
     transitionMapForCancelled.put(ActionEvent.REQUEST_CANCELLED, ActionState.CANCELLED);
     transitions.put(ActionState.CANCELLED, transitionMapForCancelled);
 
+    // RERUN
+    final Map<ActionEvent, ActionState> transitionMapForReRun = new HashMap<>();
+    transitionMapForReRun.put(ActionEvent.REQUEST_RERUN, ActionState.SUBMITTED);
```

Comment:
This is the code that needs to be tested, not some test version of i

Reply:
Had a chat with Nick and we cleared up the confusion around this so resolved